src/staking/staking_pool.h

```diff
@@ -0,0 +1,22 @@
+#ifndef ELECTRIC_CASH_STAKING_POOL_H
+#define ELECTRIC_CASH_STAKING_POOL_H
+#include "amount.h"
+
+class StakingPool {
+
+private:
+    static StakingPool* srpInstance;
+    CAmount balance = 0;
+    StakingPool(){};
```

Comment:
I think we're going to need a constructor with the balance as a parameter in near future.

Reply:
Added